pkg/controller/nodes/task/setup_ctx.go

```diff
@@ -49,7 +50,8 @@ func (n nameSpacedSetupCtx) ResourceRegistrar() pluginCore.ResourceRegistrar {
 }
 
 func (n nameSpacedSetupCtx) MetricsScope() promutils.Scope {
-	return n.SetupContext.MetricsScope().NewSubScope(n.pluginID)
+	p := strings.Replace(n.pluginID, "-", "_", -1)
```

Comment:
I don't think this is enough (looking at the check done in prometheus)... I'm fixing it in stdlib... give me a min

Reply:
thank you

Reply:
https://github.com/flyteorg/flytestdlib/pull/114
@kumare3 please take a look